javascript/react/introduction-to-hooks/why-were-hooks-needed.md

```diff
@@ -0,0 +1,81 @@
+---
+author: kapnobatai136
+
+levels:
+
+  - medium
+  
+aspects:
+
+  - new
+
+  - workout
+
+  - deep
+
+type: normal
+
+category: must-know
+
+links:
+  - '[render-props](https://reactjs.org/docs/render-props.html){documentation}'
+  - '[higher-order-components](https://reactjs.org/docs/higher-order-components.html){documentation}'
+
+---
+
+# Why were hooks needed?
+
+---
+## Content
+
+Hooks represent a new addition to React, and they allow you to use state or other useful features without using class components.
+
+Hooks were created to solve a number of issues that were encountered in the past five years since React was created.
```

Comment:
Make this sentence a bit better at standing the test of time:

```suggestion
Hooks were created to solve a number of issues that were encountered in the first five years since React was created.
```